src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 10px 0;
+}
+
+.container-logo {
+  position: relative;
+  display: flex;
+  margin: 10px 50px;
+}
+
+.container-logo img {
+  display: block;
+}
+
+nav {
+  margin-left: auto;
+}
+
+nav ul {
+  display: flex;
+  list-style-type: none;
+}
+
+nav ul li {
+  margin-right: 20px;
+}
+
+nav ul li a {
+  text-decoration: none;
+  font-size: 12px;
+  color: black;
+  display: block;
+  text-align: center;
+  height: 40px;
+  line-height: 14.6px;
+  position: relative;
+}
+
+nav ul li a.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 1px;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+}
```

Comment:
It's great that you are using pseudo-classes and pseudo-elements. Just remember to style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.